.github/workflows/build_all_apps.yml

```diff
@@ -0,0 +1,89 @@
+name: Build all Rust apps
+
+on:
+    workflow_dispatch:
+        inputs:
+            sdk_branch:
+                type: string
+                required: false
+                default: ''
+    pull_request: 
+
+
+jobs:    
+    test-build:
+        name: Build for all targets
+        strategy:
+            fail-fast: false
+            matrix:
+                include: 
+                    #- repo: 'app-mobilecoin'
+                    #  branch: 'develop'
+                    - repo: 'app-radix-babylon'
+                      branch: 'develop'
+                    - repo: 'app-boilerplate-rust'
+                      branch: 'main'
+
```

Comment:
Could be good to not have an hardcoded list but to build dynamically the list. 
I tried to do that in this [PR](https://github.com/LedgerHQ/ledger-secure-sdk/pull/553/files) for C app (name: Get C apps repos)

Reply:
yes it could be a good idea

Reply:
ok

Reply:
If nobody add it to ledgered it's never going to be there ;)